### DISCUSSION TITLE
Feature Requests.

### DISCUSSION THREAD

**Discussion by SuhasDissa** [you-apps/ClockYou#189] [Ideas]
Hi everyone,

We're starting this thread to help keep the issue log clean and focused on the most critical issues. Please use this thread to post your feature requests and design suggestions.

We appreciate your feedback!

**Comment by Grooty12** [comment]
I have a suggestion for when you are scrolling to set the time for a timer:
Make it vibrate each time you scroll past a number. This is the case for example on IOS, and makes it not only feel better to scroll, but makes it have weight to it, so that it feels more natural.

**Comment by rusty-snake** [comment]
### Stopwatch

Start and stop by taping in the circle.

**Comment by rusty-snake** [comment]
### Alarms

- Release Android 14 fix
- Work with non-UTC timezones and 24 hour clocks
- Support fullscreen notifications when device is locked.
- Stop alarms w/o force stopping the app.

**Comment by ginex25** [comment]
### Timer 

A function has already been implemented which keeps the display on as soon as the stopwatch is running, Ticket: #206 . It would be nice if this could also be implemented for the timer.
